xbmc/cores/VideoPlayer/Process/windows/ProcessInfoWin.cpp

```diff
@@ -24,7 +24,7 @@
 #include <set>
 
 // Override for platform ports
-#if defined(TARGET_WINDOWS) || defined(TARGET_WIN10)
+#if defined(TARGET_WINDOWS_DESKTOP)
```

Comment:
exclude the file for store builds in cmake instead of this ifdef
do the same for `ProcessInfoWin10.cpp`